### DISCUSSION TITLE
Not able to connect to social accounts

### DISCUSSION THREAD

**Discussion by aphandersen** [idno/idno#3137] [Q&A]
I have just installed WithKnown on my own webserver. If I read the how to guide, I should be able to connect to my social accounts by choosing Account Settings and then Connect services. 

https://withknown.com/guides/beginner/social-connections/

However, when I login and choose Account Settings, there is nothing called "Connect services":

![Account-settings](https://user-images.githubusercontent.com/10374349/201942735-7f012d0f-a08f-4b67-b833-c22ad06b4c76.jpg)

So my question is, where do I connect to social services like Twitter, Mastodon etc.?

**Comment by Cambridgeport90** [comment]
You have two potential things that you’re seeing; unlike during times of old, Known now requires plugins for each social network that you would like to post to; also, some of them are no longer active; Facebook, for instance, doesn’t have an API anymore for individual profiles, instead, if you want to post to it, you have to first create a page. Twitter… for now, still works, and most of the others do, too. Instagram, though, might be broken, if learning about other applications trying to post to it finding out they can’t.

From: Andreas Andersen ***@***.***>
Sent: Tuesday, November 15, 2022 09:23
To: idno/known ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [idno/known] Not able to connect to social accounts (Discussion #3137)


I have just installed WithKnown on my own webserver. If I read the how to guide, I should be able to connect to my social accounts by choosing Account Settings and then Connect services.

https://withknown.com/guides/beginner/social-connections/

However, when I login and choose Account Settings, there is nothing called "Connect services":

[Account-settings]<https://user-images.githubusercontent.com/10374349/201942735-7f012d0f-a08f-4b67-b833-c22ad06b4c76.jpg>

So my question is, where do I connect to social services like Twitter, Mastodon etc.?

**Reply by aphandersen** [reply]
Okay i found a plugin for both Twitter and Mastodon, and extracted the folders to IdnoPlugins. They are now visible in my plugins list, but nothing happens when I press Enable. 

Do plugins need to be registered other places than just extacting them to the plugin folder?

**Comment by Cambridgeport90** [comment]
I don’t think so… You’re on the right track. I’m going to let the more plugin developer oriented members of the community speak up. I’m more admin than dev at the moment.
